modules/search-service-impl/src/main/java/org/opencastproject/feed/impl/FeedServiceImpl.java

```diff
@@ -94,6 +97,34 @@
   /** The security service */
   private SecurityService securityService = null;
 
+  /*
+   *
+   * TEST
+   * /feeds/listFeedServices
+   *
+   */
+
+  @GET
+  @Path("/listFeedServices")
+  @Produces(MediaType.APPLICATION_JSON)
+  public String listFeedServices() {
+
+    Gson gson = new Gson();
```

Comment:
Gson is thread-safe. Please avoid creating per-request instances but rather re-use a static instance.